schemas/v1/config/modelling.json

```diff
@@ -0,0 +1,112 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/imperialCHEPI/healthgps/main/schemas/v1/config/modelling.json",
+    "type": "object",
+    "properties": {
+        "ses_model": {
+            "type": "object",
+            "properties": {
+                "function_name": {
+                    "type": "string"
+                },
+                "function_parameters": {
+                    "type": "array",
+                    "items": {
+                        "type": "number"
+                    }
```

Comment:
Just checking this **array** doesn't require a minimum number of values? Same applies for other arrays in this schema I guess.

Reply:
I'm not sure... @jamesturner246?

Reply:
I think these depend on the SES model -- e.g. normal has mean, std. Maybe there are other models that have none? Never uses the SES module.

Reply:
Hmm... ok. I'll leave as is for now. We can always add extra constraints in future if needed.